packages/mobile/src/analytics/Properties.tsx

```diff
@@ -495,6 +495,7 @@ interface SendEventsProperties {
   }
   [SendEvents.send_confirm_back]: undefined
   [SendEvents.send_confirm_send]: {
```

Comment:
What do you think about adding a `isFromDeepLink` property?

Reply:
Yep! I opened #6564 to track this.